MAUI/Barcode/two-dimensional-symbology.md

```diff
@@ -0,0 +1,273 @@
+platform: maui
+control: SfBarcodeGenerator
+documentation: ug
+---
+
+# Two-dimensional symbology in .NET MAUI Barcodes (SfBarcodeGenerator)
+
+Two dimensional Barcode is a way to represent information by using the two-dimensional approach. It is similar to one dimensional Barcode, but can represent more data per unit area. The barcode generator control supports the following two dimensional symbology:
+
+* `QR Code`
+* `Data Matrix`
+
+## QR Code
+
+A `QR Code` is a two-dimensional barcode that consists of a grid of dark and light dots or blocks that form a square. The data encoded in the barcode can be numeric, alphanumeric, or Shift JIS characters.
+
+* The QR Code uses version from 1 to 40. Version 1 measures 21 modules x 21 modules, Version 2 measures 25 modules x 25 modules, and so on. The number of modules increases in steps of 4 modules per side up to Version 40 that measures 177 modules x 177 modules.
+* Each version has its own capacity. By default, the barcode control automatically sets the version according to the length of the input text.
+* The QR Barcodes are designed for industrial uses and also commonly used in consumer advertising.
+
+{% tabs %}
+
+{% highlight xaml hl_lines="6" %}
+
+   <barcode:SfBarcodeGenerator Value="www.syncfusion.com"
+                               ShowText="True"
+                               HeightRequest="300"
+                               WidthRequest="300">
+        <barcode:SfBarcodeGenerator.Symbology>
+            <barcode:QRCode />
+        </barcode:SfBarcodeGenerator.Symbology>
+    </barcode:SfBarcodeGenerator>
+
+{% endhighlight %}
+
+{% highlight C# hl_lines="5" %}
+
+    SfBarcodeGenerator barcode = new SfBarcodeGenerator();
+    barcode.HeightRequest = 300;
+    barcode.WidthRequest = 300;
+    barcode.Value = "www.syncfusion.com";
+    barcode.Symbology = new QRCode();
+    barcode.ShowText = true;
+    this.Content = barcode;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![.NET MAUI Barcode Generator QR Code Symbology](images/two-dimensional-symbology/maui-qr-code-symbology.png)
+
+The data that can be stored in the QR code depends upon the following property:
+
+* `Error correction level`
+* `QR code version`
+* `Input mode`
+
+As like one dimensional symbology, the two dimensional symbology also supports the `Module` property.
+The property is used to define the size of the smallest line or dot of the barcode. If this property is not set, the size of the smallest dot of the barcode is calculated based on the available size.
+
+### Error correction level
+
+The `ErrorCorrectionLevel` property employs error correction to generate a series of error correction code words that are added to the data code word sequence in order to enable the symbol to withstand damage without loss of data. By default, its value is set as `Auto`.
+
+Low - it recovers the data up to 7%
+Medium - it recovers the data up to 15%
+Quartile - it recovers the data up to 25%
```

Comment:
Quartile - it recovers the data up to 25%.

Reply:
Changes updated.